hedera-node/src/main/java/com/hedera/services/legacy/core/jproto/JContractIDKey.java

```diff
@@ -59,6 +60,10 @@ public JContractIDKey(long shardNum, long realmNum, long contractNum) {
 		this.contractNum = contractNum;
 	}
 
+	public static JContractIDKey fromId(final Id source) {
```

Comment:
nit: Java Docs

Reply:
We need to add unit tests for the changes in this file

Reply:
Done, code cov increased from 25 to 100 % as well.